packages/bsky/src/views/index.ts

```diff
@@ -132,10 +134,14 @@ export class Views {
     }
   }
 
-  profile(did: string, state: HydrationState): ProfileView | undefined {
+  profile(
+    did: string,
+    state: HydrationState,
+    includeKnownFollowers = false,
```

Comment:
yeah similarly to the hydrator, we could remove the parameter on this, and only call `knownFollowers(did, state) from within the `profileDetailed()` view, then just munge that into profile viewer state